### PR TITLE
en: improve the doc of tidb-lightning

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added, or deleted? (Required)

Improve the doc of tidb-lightning about restoring data 

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/1217
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/restore-data-using-tidb-lightning.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Use the following command to get the default configuration of TiDB Lightning:
 helm inspect values pingcap/tidb-lightning --version=${chart_version} > tidb-lightning-values.yaml
 ```
 
-Configure a `backend` used by TiDB Lightning according to your needs. The options include `importer`, `local`, and `tidb`.
+Configure a `backend` used by TiDB Lightning depending on your needs. To do that, you need to set the `backend` value in `values.yaml` to an option in `importer`, `local`, or `tidb`.
```
Comment (qiancai) [inline en/restore-data-using-tidb-lightning.md:97]:
```suggestion
Configure a `backend` used by TiDB Lightning depending on your needs. To do that, you can set the `backend` value in `values.yaml` to an option in `importer`, `local`, or `tidb`.
```

File: en/restore-data-using-tidb-lightning.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ Use the following command to get the default configuration of TiDB Lightning:
 helm inspect values pingcap/tidb-lightning --version=${chart_version} > tidb-lightning-values.yaml
 ```
 
-Configure a `backend` used by TiDB Lightning according to your needs. The options include `importer`, `local`, and `tidb`.
+Configure a `backend` used by TiDB Lightning depending on your needs. To do that, you can set the `backend` value in `values.yaml` to an option in `importer`, `local`, or `tidb`.
```
Comment (qiancai) [inline en/restore-data-using-tidb-lightning.md:97]:
```suggestion
In `values.yaml`, configure a `backend` used by TiDB Lightning depending on your needs: `importer`, `local`, or `tidb`.
```

**Comment by qiancai** [comment]
@CharLotteiu PTAL. Thanks.

File: en/restore-data-using-tidb-lightning.md [hunk 3]
```diff
@@ -6,15 +6,15 @@ aliases: ['/docs/tidb-in-kubernetes/dev/restore-data-using-tidb-lightning/']
...
 
-TiDB Lightning supports [three backends](https://docs.pingcap.com/tidb/stable/tidb-lightning-backends): `importer`, `local`, and `tidb`.
+TiDB Lightning supports three backends: `Importer-backend`, `Local-backend`, and `TiDB-backend`. For the differences of these backends and how to choose backends, see [TiDB Lightning Backends](https://docs.pingcap.com/tidb/stable/tidb-lightning-backends).
 
-- For the `importer` backend, both tikv-importer and tidb-lightning need to be deployed.
-- For the `local` or `tidb` backend, only tidb-lightning needs to be deployed.
-- For the `tidb` backend, it is recommended to import data using CustomResourceDefinition (CRD) in TiDB Operator v1.1 and later versions. For details, refer to [Restore Data from GCS Using TiDB Lightning](restore-from-gcs.md) or [Restore Data from S3-Compatible Storage Using TiDB Lightning](restore-from-s3.md)
+- For `Importer-backend`, both tikv-importer and tidb-lightning need to be deployed.
+- For `Local-backend` or `TiDB-backend`, only tidb-lightning needs to be deployed.
```
Comment (CharLotteiu) [inline en/restore-data-using-tidb-lightning.md:16]: How about separating the description of `Local-backend` and `TiDB-backend` here, and organizing the list like this:
- For `Importer-backend`, xxxxx
- For `Local-backend`, xxxxx
- For `TiDB-backend`, only tidb-lightning needs to be deployed and it is recommended to .....

One entry for each backend.

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: en/restore-data-using-tidb-lightning.md (hunk 3 above)
Comment (qiancai) [inline en/restore-data-using-tidb-lightning.md:16]:
```suggestion
- For `Local-backend`, only tidb-lightning needs to be deployed.
```

**Comment by qiancai** [comment]
@CharLotteiu PTAT again. Thanks.

**Review by CharLotteiu — APPROVED** [review]
LGTM

**Pull request merged**